internal/test/e2e/plugins.go

```diff
@@ -72,12 +72,15 @@ var VersionCompatibilityOSSPluginConfigurationTests = []VersionCompatibilityPlug
 	//   - if both 'aws_region' and 'host' are set
 	//     just drop 'host' and keep 'aws_region'
 	//     since these used to be  mutually exclusive
+	//   - remove 'aws_assume_role_arn' and 'aws_role_session_name'
 	{
 		Name: "aws-lambda",
 		Config: `{
 			"aws_region": "AWS_REGION",
 			"host": "192.168.1.1",
-			"function_name": "FUNCTION_NAME"
+			"function_name": "FUNCTION_NAME",
+			"aws_assume_role_arn": "foo",
+			"aws_role_session_name": "kong"
```

Comment:
in some PRs, we added also tests for a 3.0 config and make sure fields were passed (but I also see we don´t do it in all PRs) Should we follow this and add an aws plugin config with these fields in a `...After3.0` test?

Reply:
Good call. Added it here https://github.com/Kong/koko/pull/532/commits/f51b41f1e15b0282e989831083274e16843b9267